### PR TITLE
Improve footer 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, there is no logo in the footer. #10 
- Currently, there is a missing line in the footer. #15 

**What is the new behavior (if this is a feature change)?**
- Our logo has been added to the footer. #10 
- The missing line has been added to the footer. #15

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks fine to me

File: src/components/molecules/Footer/index.jsx [hunk 1]
```diff
@@ -111,8 +113,8 @@ class Footer extends React.Component{
                 <MDBContainer className="text-center text-md-left pt-4">
                     <MDBRow>
                         <MDBCol md="2">
-                            <h5>Logo</h5>
-                            <p>
+                            <img src={Logo} style={{ height: "2.5rem" }} />
```
Comment (kleberbaum) [inline src/components/molecules/Footer/index.jsx]: Is it absolutely necessary to define this style in html?

File: src/components/molecules/Footer/index.jsx [hunk 2]
```diff
@@ -157,20 +159,20 @@ class Footer extends React.Component{
                             <hr className="agency-red mb-4 mt-0 d-inline-block" />
                             <ul>
                             <li className="list-unstyled">
-                                <a href="#!"><MDBIcon icon="home" />Villach, Kärnten, AT</a>
+                                <MDBIcon icon="home" />Villach, Kärnten, AT
                             </li>
                             <li className="list-unstyled">
-                                <a href="#!"><MDBIcon far icon="envelope" />info@aichner-christian.com</a>
+                                <a href="mailto:info@aichner-christian.com"><MDBIcon far icon="envelope" />info@aichner-christian.com</a>
```
Comment (kleberbaum) [inline src/components/molecules/Footer/index.jsx]: Unnecessarily long line.
We allow up to 119 characters as this is the width of GitHub code review.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
It should be ok to merge after a few changes.

File: src/components/molecules/Footer/index.jsx (hunk 1 above)
Comment (Aichnerc) [reply src/components/molecules/Footer/index.jsx]: Nah, I just copied it from the MDB Pro documentation. But you're right, it's way cleaner to define it in footer.scss.

File: src/components/molecules/Footer/index.jsx (hunk 2 above)
Comment (Aichnerc) [reply src/components/molecules/Footer/index.jsx]: The entire file has been cleaned up regarding line length.

**Review by Aichnerc — COMMENTED** [review]
Everything looks fine to me

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**